WcaOnRails/config/application.rb

```diff
@@ -73,5 +73,8 @@ class Application < Rails::Application
     config.middleware.use Middlewares::WardenUserLogger, logger: ->(s) { Rails.logger.info(s) }
 
     config.autoload_paths << Rails.root.join('lib')
+
+    config.action_controller.asset_host = ENVied.ROOT_URL
+    Rails.application.routes.default_url_options[:host] = ENVied.ROOT_URL
```

Comment:
What's this for? I thought Rails would deduce this from the incoming HTTP HOST header. Although in general this is probably useful for cases like background jobs that don't have corresponding HTTP requests.